.circleci/config.yml

```diff
@@ -1,88 +1,20 @@
-e2eTest: &e2eTest
-    machine: true
-    working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/apprclient
-    steps:
-    - checkout
-
-    - attach_workspace:
-        at: .
-
-    - run:
-        name: Get e2ectl binary
-        command: |
-          wget -q $(curl -sS https://api.github.com/repos/giantswarm/e2ectl/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
-          chmod +x ./e2ectl
-
-    - run:
-        name: Init cluster
-        command: |
-          ./e2ectl cluster create
-          cp $(./e2ectl kubeconfig path) ${E2E_TEST_DIR}/kubeconfig
-
-    - run:
-        name: Run test
-        command: |
-          docker run --rm --network host \
-            -v $(pwd)/${E2E_TEST_DIR}:/e2e \
-            -v $(pwd):/go/src/github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} \
-            -e E2E_KUBECONFIG=/e2e/kubeconfig \
-            -e CIRCLE_SHA1=${CIRCLE_SHA1} \
-            -w /go/src/github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} \
-            golang:1 go test -v -tags k8srequired ./${E2E_TEST_DIR}/
-
-    - run:
-        name: Export logs
-        command: |
-          ./e2ectl logs export --logs-dir=kind/logs
-        when: always
-
-    - store_artifacts:
-        path: kind/logs
-        destination: logs
-
-    - run:
-        name: Finish with cleanup, no matter if the test succeeded or not
-        command: ./e2ectl cluster delete
-        when: always
-
-
-
-version: 2
-jobs:
-  build:
-    machine: true
-    steps:
-    - checkout
-
-    - run: |
-        wget -q $(curl -sS -H "Authorization: token $RELEASE_TOKEN" https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
-        chmod +x ./architect
-        ./architect version
-
-    - run: ./architect build
-
-    - store_test_results:
-        path: /tmp/results
-
-    - deploy:
-        command: |
-          if [ "${CIRCLE_BRANCH}" == "master" ]; then
-            ./architect deploy
-          fi
-
-
-
-  e2eTestBasic:
-    environment:
-      E2E_TEST_DIR: "integration/test/basic"
-    <<: *e2eTest
+version: 2.1
 
+orbs:
+  architect: giantswarm/architect@0.8.4
 
 workflows:
-  version: 2
-  build_e2e:
+  test:
     jobs:
-      - build
-      - e2eTestBasic:
+      - architect/go-test:
+          name: go-test
+          filters:
+            # Trigger job also on git tag.
+            tags:
+              only: /^v.*/
+
+      - architect/integration-test:
+          name: integration-test-basic
+          test-dir: "integration/test/basic"
```

Comment:
@rossf7 do you know what is missing in the integration test here? 

```
=== RUN   Test_Client_GetReleaseVersion
2020/03/25 13:17:58 waiting for server at 15s: {"kind":"executionFailedError","annotation":"expected 1 pod got 0","stack":[{"file":"/home/circleci/.go_workspace/src/github.com/giantswarm/apprclient/integration/test/basic/basic_test.go","line":125}]}
2020/03/25 13:18:13 waiting for server at 15s: {"kind":"executionFailedError","annotation":"expected 1 pod got 0","stack":[{"file":"/home/circleci/.go_workspace/src/github.com/giantswarm/apprclient/integration/test/basic/basic_test.go","line":125}]}
--- FAIL: Test_Client_GetReleaseVersion (30.08s)
    basic_test.go:93: could not push tarball open /e2e/fixtures/tb-chart.tar.gz: no such file or directory
    basic_test.go:78: could not delete release unknown status error
FAIL
FAIL	github.com/giantswarm/apprclient/integration/test/basic	30.196s
```

See https://circleci.com/gh/giantswarm/apprclient/258.

Reply:
Does that need some mounting for the fixtures? How is that done with the latest advances here?

Reply:
Yes I think this `/e2e/` dir is some e2e-harness magic.

Its used here when the fixture gets created.
https://github.com/giantswarm/apprclient/blob/ee23332dfe5c408f0e11b9a86d60edfa78a6b2ac/integration/test/basic/basic_test.go#L91

I also tried this in https://github.com/giantswarm/apprclient/pull/36. I'll have a go at this now. I'll let you know.